templates/search/blocks/search_results__list-card--grid.html

```diff
@@ -10,13 +9,12 @@
 {% endif %}
 
 {% comment %} TODO downloadable, and record variables {% endcomment %}
-{% load search_tags records_tags static %}
 
 <li class="search-results__list-card--grid">
-    {% if record|record_detail:"digitised" %}
+    {% if record.is_digitised %}
```

Comment:
The record|record_detail:"digitised" [implementation is removed.](https://github.com/nationalarchives/ds-wagtail/pull/422/files#diff-04206e540f8f7fda4cf059782e16ee16e502465b2ac29d463c00238a146b504dL19-L29)
We could add it to [current model implementation](https://github.com/nationalarchives/ds-wagtail/blob/19df21c949fde726b210e9da49f732a4442fea38/etna/records/models.py#L206) to so that it shows up in search results.

Reply:
@TNA-Allan According to the spec, `digitised` should be present as a boolean at the top level, which is where the `is_digitised` looks. I would expect that's true in a 'search results' context too.

Reply:
Have played around, and it looks like there is no 'digitised' value available ANYWHERE in result content when specifying `template=details` (like we appear to be doing [here](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/search/views.py#L200) and [here](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/search/views.py#L401)), so I guess this may have never worked!

Reply:
Check with this one on develop  /search/catalogue/?group=digitised

Reply:
Ah, so what the template includes is also dependant on the filters applied to the search? That's 'fun' :D

Reply:
Resolved in c1db32a :)